src/main/scala/is/hail/asm4s/FunctionBuilder.scala

```diff
@@ -242,6 +239,13 @@ class FunctionBuilder[F >: Null](val parameterTypeInfo: Array[MaybeGenericTypeIn
     val sw1 = new StringWriter()
     var bytes: Array[Byte] = new Array[Byte](0)
     try {
+      for (method <- cn.methods.asInstanceOf[util.List[MethodNode]].asScala) {
+        val count = method.instructions.size
```

Comment:
so this counts the number of instructions in a method, and not the actual bytesize of the method, and I had this whole comment typed out before I noticed that one of the comments on the stackoverflow thread says that the `DontCompileHugeMethods` flag counts instructions, as well! so this is probably better than counting the bytes, right?

I do have one minor nitpick on that note, though---can you say `instruction count` or something instead of `bytecode size` so we don't confuse ourselves in the future in case the distinction becomes more relevant?

Reply:
mostly, I keep confusing myself about this. but if it's not confusing to anyone else then it also looks fine as is.